src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -172,7 +172,7 @@ javac.opt.arg.Xlint=\
 javac.opt.Xlint.custom=\
     Warnings to enable or disable, separated by comma.\n\
     Precede a key by '-' to disable the specified warning.\n\
-    Supported keys are:
+    Use option --help-lint to see the supported keys.
```

Comment:
Suggest removing the word "option".
This does not affect the CSR.

Reply:
Fixed.